OverlayPlugin.Core/MemoryProcessors/InCombat/LineInCombat.cs

```diff
@@ -66,16 +66,24 @@ public void Update()
             // always set this to true if lastEventArgs == null so that if ACT is started while out of combat,
             // the overlays are hidden/shown appropriately.
             bool inGameCombatChanged = lastEventArgs == null ? inGameCombat : lastEventArgs.InGameCombat != inGameCombat;
+
+            // Add two boolean variables into the logline to indicate exactly which part was changed.
+            // Useful if a plugin only cares about the change of in-game or ACT combat state to trigger other events.
+            bool isACTChanged = lastEventArgs != null && lastEventArgs.InACTCombat != inACTCombat;
+            bool isGameChanged = lastEventArgs != null && lastEventArgs.InGameCombat != inGameCombat;
```

Comment:
Sorry for one more last thought here, but I wonder if it would be better to always set these to true if there aren't previous events. That way the first time the line is written, it will always say that both have changed. Does that make more sense for the kinds of things you're doing with log line? The only reason it isn't this way now is because of that `// TODO: backwards-compatible` paragraph above.

(Also one final cleanup thought is to maybe rename `inGameCombatChanged` to `inGameCombatChangedBackwardsCompat` or something which is kind of long but having both `isGameChanged` and `inGameCombatChanged` variables next to each other is a bit confusing. Could you also group together these additions and the WriteLine up above the `// TODO: backwards-compatible etc...` line so that the event logic and the write line logic aren't intermingled?)